src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,169 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capacity;
+    private int threshold;
+    private int size;
+    private double loadFactor;
```

Comment:
this should be constant

Reply:
I don't understand...because these variables are used in all methods...